src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputViolation.java

```diff
@@ -26,6 +26,15 @@
     /** Pattern to match the symbol: "{". */
     private static final Pattern OPEN_CURLY_PATTERN = Pattern.compile("\\{");
 
+    /** Pattern to match the symbol: "(". */
```

Comment:
how do you think, there will be more such cases?

it seems like we replacing `\\symbol` to `\\\\symbol`, maybe we should generalize this logic